src/main/resources/mybatis/features.xml

```diff
@@ -36,20 +36,35 @@
 
      <sql id="selectFeaturesQuery">
           (select
-              'monitoring-locations' collection_id, #{monLocTitle,jdbcType=VARCHAR} collectionTitle,
+              'monitoring-locations' collection_id, #{monLocTitle,jdbcType=VARCHAR} collection_title,
```

Comment:
For the features query we should always have a single collection id. Use that to our advantage and do not have this union query with the hard to read, interpret, and duplicated login run on UNION